#global testerror "Err0389"

#test
{
    let a: [4],
}